storybook/cypress.config.ts

```diff
@@ -5,6 +5,20 @@ export default defineConfig({
   e2e: {
     setupNodeEvents(on, _config) {
       addMatchImageSnapshotPlugin(on);
+      on("task", {
+        log(message) {
```

Comment:
Wellicht deze naar "logError", of `console.log` gebruiken?

Of wellicht `log(message, logLevel: "error" | "log" = "log")` (of iets met "info")

Reply:
Het is `error` met `console.error` geworden

Reply:
alrighty